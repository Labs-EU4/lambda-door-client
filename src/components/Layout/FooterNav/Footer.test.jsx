import React from 'react';
import * as rtl from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Footer from './FooterNav';

describe('Footer Test', () => {
  it('renders correctly', () => {
    expect(rtl.render(<Footer />).baseElement).toMatchSnapshot();
  });

  it('shows the copy right tag', () => {
    const copyRight = rtl.render(<Footer />).queryByText(/2020 LambdaDoor/i);
    expect(copyRight).toBeInTheDocument();
  });
});
