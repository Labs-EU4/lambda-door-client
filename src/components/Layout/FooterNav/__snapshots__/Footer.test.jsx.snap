// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Test renders correctly 1`] = `
<body>
  <div>
    <div
      class="sc-bdVaJa kpDyBP"
    >
      <div
        class="tagline"
      >
        Built by Lambda students, for Lambda students.
      </div>
      <div
        class="copy-right"
      >
         Â© 2020 LambdaDoor
      </div>
      <div
        class="social-handle"
      >
        <a
          class="twitter"
          href="https://twitter.com/LambdaSchool"
        >
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-twitter fa-w-16 fa-2x "
            data-icon="twitter"
            data-prefix="fab"
            focusable="false"
            role="img"
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
              fill="currentColor"
            />
          </svg>
        </a>
        <a
          class="facebook"
          href="https://www.facebook.com/LambdaSchoolOnline/"
        >
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-facebook fa-w-16 fa-2x "
            data-icon="facebook"
            data-prefix="fab"
            focusable="false"
            role="img"
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M504 256C504 119 393 8 256 8S8 119 8 256c0 123.78 90.69 226.38 209.25 245V327.69h-63V256h63v-54.64c0-62.15 37-96.48 93.67-96.48 27.14 0 55.52 4.84 55.52 4.84v61h-31.28c-30.8 0-40.41 19.12-40.41 38.73V256h68.78l-11 71.69h-57.78V501C413.31 482.38 504 379.78 504 256z"
              fill="currentColor"
            />
          </svg>
        </a>
        <a
          class="linkdIn"
          href="https://www.linkedin.com/school/lambdaschool/"
        >
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-linkedin fa-w-14 fa-2x "
            data-icon="linkedin"
            data-prefix="fab"
            focusable="false"
            role="img"
            viewBox="0 0 448 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
              fill="currentColor"
            />
          </svg>
        </a>
        <a
          class="webpage"
          href="https://lambdaschool.com/"
        >
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-globe fa-w-16 fa-2x "
            data-icon="globe"
            data-prefix="fas"
            focusable="false"
            role="img"
            viewBox="0 0 496 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M336.5 160C322 70.7 287.8 8 248 8s-74 62.7-88.5 152h177zM152 256c0 22.2 1.2 43.5 3.3 64h185.3c2.1-20.5 3.3-41.8 3.3-64s-1.2-43.5-3.3-64H155.3c-2.1 20.5-3.3 41.8-3.3 64zm324.7-96c-28.6-67.9-86.5-120.4-158-141.6 24.4 33.8 41.2 84.7 50 141.6h108zM177.2 18.4C105.8 39.6 47.8 92.1 19.3 160h108c8.7-56.9 25.5-107.8 49.9-141.6zM487.4 192H372.7c2.1 21 3.3 42.5 3.3 64s-1.2 43-3.3 64h114.6c5.5-20.5 8.6-41.8 8.6-64s-3.1-43.5-8.5-64zM120 256c0-21.5 1.2-43 3.3-64H8.6C3.2 212.5 0 233.8 0 256s3.2 43.5 8.6 64h114.6c-2-21-3.2-42.5-3.2-64zm39.5 96c14.5 89.3 48.7 152 88.5 152s74-62.7 88.5-152h-177zm159.3 141.6c71.4-21.2 129.4-73.7 158-141.6h-108c-8.8 56.9-25.6 107.8-50 141.6zM19.3 352c28.6 67.9 86.5 120.4 158 141.6-24.4-33.8-41.2-84.7-50-141.6h-108z"
              fill="currentColor"
            />
          </svg>
        </a>
      </div>
    </div>
  </div>
</body>
`;
